java/client/src/org/openqa/selenium/OutputType.java

```diff
@@ -24,8 +24,7 @@
 import java.util.Base64;
 
 /**
- * Defines the output type for a screenshot. See org.openqa.selenium.Screenshot for usage and
- * examples.
+ * Defines the output type for a screenshot.
```

Comment:
There's already a `@see TakesScreenshot` down below: I think it would be redundant.